Userland/Libraries/LibJS/Runtime/ArrayBuffer.h

```diff
@@ -43,6 +43,8 @@ class ArrayBuffer : public Object {
     void set_detach_key(Value detach_key) { m_detach_key = detach_key; }
 
     void detach_buffer() { m_buffer = Empty {}; }
+
+    // 25.1.2.2 IsDetachedBuffer ( arrayBuffer ), https://tc39.es/ecma262/#sec-isdetachedbuffer
```

Comment:
This also needs inline comments for step 1 and 2.

Reply:
Wil do!